### DISCUSSION TITLE
scp without hostkey save from the client

### DISCUSSION THREAD

**Discussion by gpointer** [drakkan/sftpgo#725] [Q&A] [answered]
Hi Drakkan,
i copied all keys under /etc/ssh to the new data-exchange-Server sftpgo to the same directory and under /etc/sftpgo with the names you give. 

-rw-------   1 sftpgo sftpgo   227 Feb  9 08:21 id_ecdsa
-rw-r--r--   1 sftpgo sftpgo   183 Feb  9 08:21 id_ecdsa.pub
-rw-------   1 sftpgo sftpgo   411 Feb  9 08:21 id_ed25519
-rw-r--r--   1 sftpgo sftpgo   103 Feb  9 08:21 id_ed25519.pub
-rw-------   1 sftpgo sftpgo  1675 Feb  9 08:21 id_rsa
-rw-r--r--   1 sftpgo sftpgo   403 Feb  9 08:21 id_rsa.pub

But if i make a transmission it wants an input like this
The authenticity of host '[sshgate-neu]:44447 ([10.51.201.99]:44447)' can't be established.
ECDSA key fingerprint is SHA256:Rwzkglnu+xczrOnYpn6nST6z6e0h3d+w0vPT2kQQ5jM.
Are you sure you want to continue connecting (yes/no)?

but it should not do this , because of the old hostkeys. i need a transmission without input from the client because of the automatic jobs.
Do you have an idea what is my mistake?
Thank you.

**Comment by drakkan** [comment, marked as answer]
Hi,

this is a client side issue, you have to disable host key verification. Take a look [here](https://serverfault.com/questions/330503/scp-without-known-hosts-check)

**Reply by gpointer** [reply]
Thank you for quick answer :-)
